src/Microsoft.Health.Dicom.Api/Features/Filters/DicomRequestContextRouteDataPopulatingFilterAttribute.cs

```diff
@@ -15,7 +15,7 @@
 namespace Microsoft.Health.Dicom.Api.Features.Filters
 {
     [AttributeUsage(AttributeTargets.Class)]
-    public class DicomRequestContextRouteDataPopulatingFilterAttribute : ActionFilterAttribute
+    public sealed class DicomRequestContextRouteDataPopulatingFilterAttribute : ActionFilterAttribute
```

Comment:
breaking change?

Reply:
Yup, this is breaking along with some other access modifier changes